### ISSUE TITLE
Proper validation/ message display on respective fields (User Login)

### ISSUE THREAD

**Issue by deleenananaa** [Tenzing07/One-Byte-Food#7]
To enhance user experience and accessibility, validations and error handling for user login should be displayed in a popup or dialogue box within the system. The bug is shown in the screenshot below:
![image](https://github.com/Tenzing07/One-Byte-Food/assets/119421004/b942bcb8-f963-4711-b179-612e9af83f91)

**Comment by SamirShresthaa** [comment]
I have added the popup in the user login to enhance user experience and accessibility, validations and error handling
